packages/zero-client/src/client/zero.ts

```diff
@@ -957,7 +957,7 @@ export class Zero<
    * longer query or mutate data with it, and its query views stop updating.
    */
   get closed(): boolean {
-    return this.#rep.closed;
+    return this.#connectionManager.is(ConnectionStatus.Closed);
```

Comment:
My only concern here is if Replicache closes the instance... but that doesn't happen.